irc/accounts.go

```diff
@@ -955,6 +966,130 @@ func (am *AccountManager) SARegister(account, passphrase string) (err error) {
 	return
 }
 
+func (am *AccountManager) NsSendpass(client *Client, accountName string) (err error) {
+	config := am.server.Config()
+	if !(config.Accounts.Registration.EmailVerification.Enabled && config.Accounts.Registration.EmailVerification.PasswordReset.Enabled) {
+		return errFeatureDisabled
+	}
+
+	account, err := am.LoadAccount(accountName)
+	if err != nil {
+		return err
+	}
+	if !account.Verified {
+		return errAccountUnverified
+	}
+	if account.Suspended != nil {
+		return errAccountSuspended
+	}
+	if account.Settings.Email == "" {
+		return errValidEmailRequired
+	}
+
+	// TODO: touch the registration throttle here?
```

Comment:
There should be some throttling, since otherwise a single IP could request password resets for everyone.

Reply:
Oh, good point, this can just touch the login throttle.